.travis.yml

```diff
@@ -0,0 +1,32 @@
+# Config file for automatic testing at travis-ci.org
+# This file will be regenerated if you run travis_pypi_setup.py
+
+language: python
+python: 3.5
+
+env:
+  - TOXENV=py35
+  - TOXENV=py34
+  - TOXENV=py33
+  - TOXENV=py27
```

Comment:
This will need some customization - I think it's possible to just start with 3.4 and 3.5.. the others are by all knowledge very rarely used since only very few Linux distros has ever shipped them.